lts-backports-changelog.rb

```diff
@@ -22,11 +27,23 @@
 
 weekly_changelog_file = ARGV[1]
 
-backports_str = `set -o pipefail ; curl --fail -u #{curl_auth} -X POST --data '{"startAt":0, "maxResults":1000,"fields": ["key"], "jql": "labels = #{lts}-fixed" }' -H "Content-Type: application/json" https://issues.jenkins-ci.org/rest/api/2/search | jq -r '.issues[] | .key'`
+command = "set -o pipefail ; curl --fail -u #{curl_auth} -X POST --data '{\"startAt\":0, \"maxResults\":1000,\"fields\": [\"key\"], \"jql\": \"labels = #{lts}-fixed\" }' -H \"Content-Type: application/json\" https://issues.jenkins-ci.org/rest/api/2/search | jq -r '.issues[] | .key'"
```

Comment:
I had to remove the `set -o pipefail` in my usage of the code, but did not spend time to investigate why my environment is different than other environments.  I was running on a Ubuntu 18.04 Linux machine (if that provides any hints why my machine behaved differently).  I believe I was using the program outside a docker container, not running it from a docker container.

No objection from me if this is left exactly as it is.  I will eventually make time enough to diagnose why my machine could not call `set -o pipefail`.

```suggestion
command = "set -o pipefail ; curl --fail -u #{curl_auth} -X POST --data '{\"startAt\":0, \"maxResults\":1000,\"fields\": [\"key\"], \"jql\": \"labels = #{lts}-fixed\" }' -H \"Content-Type: application/json\" https://issues.jenkins-ci.org/rest/api/2/search | jq -r '.issues[] | .key'"
```

Reply:
> No objection from me if this is left exactly as it is. I will eventually make time enough to diagnose why my machine could not call set -o pipefail.

Likely your default shell is not Bash